spec/spec_helper.rb

```diff
@@ -1,3 +1,10 @@
+ENV["RAILS_ENV"] ||= 'test'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.